docs/cadence/language/access-control.md

```diff
@@ -205,3 +219,199 @@ some.f[3] = 1
+and thus can only get an unauthorized reference to `SubResource` that lets them call `foo`. 
+However, someone with a `OuterEntitlement`-authorized refererence to the `OuterResource` can call the `getEntitledRef` function, 
+giving them a `SubEntitlement`-authorized reference to `SubResource` that allows them to call `bar`.
+
+This pattern is functional, but it is unfortunate that we are forced to "duplicate" the accessors to `SubResource`, 
+duplicating the code and storing two functions on the object, 
+essentially creating two different views to the same object that are stored as different functions. 
+To avoid necessitating this duplication, we add support to the language for "entitlement mappings", 
+a way to declare statically how entitlements are propagated from parents to child objects in a nesting hierarchy. 
+So, the above example could be equivalently written as:
+
+```cadence
+entitlement OuterEntitlement
+entitlement SubEntitlement
+
+// specify a mapping for entitlements called `Map`, which defines a function
+// from an input set of entitlements (called the domain) to an output set (called the range or the image)
+entitlement mapping Map {
+    OuterEntitlement -> SubEntitlement
+}
+
+resource SubResource {
+    access(all) fun foo() { ... }
+    access(SubEntitlement) fun bar() { ... }
+}
+
+resource OuterResource {
+    access(self) let childResource: @SubResource
+
+    // by referering to `Map` here, we declare that the entitlements we receive when accessing the `getRef` function on this resource
+    // will depend on the entitlements we possess to the resource during the access. 
+    access(Map) fun getRef(): auth(Map) &SubResource {
+        return &self.childResource as auth(Map) &SubResource
+    }
+
+    init(r: @SubResource) {
+        self.childResource = r
+    }
+}
+
+// given some value `r` of type `@OuterResource`
+let pubRef = &r as &OuterResource
+let pubSubRef = pubRef.getRef() // has type `&SubResource`
+
+let entitledRef = &r as auth(OuterEntitlement) &OuterResource
+let entitledSubRef = entitledRef.getRef() // `OuterEntitlement` is defined to map to `SubEntitlement`, so this access yields a value of type `auth(SubEntitlement) &SubResource`
+Entitlement
+
+// `r` is an owned value, and is thus considered "fully-entitled" to `OuterResource`,
+// so this access yields a value authorized to the entire image of `Map`, in this case `SubEntitlement`
+let alsoEntitledSubRef = r.getRef() 
+```
+
+{/* TODO: Update once mappings can be used on regular composite fields */}
+
+Entitlement mappings may be used either in accessor functions (as in the example above), or in fields whose types are references. Note that this latter
+usage will necessarily make the type of the composite non-storage, since it will have a reference field. 
+
+{/* TODO: once the Account type refactor is complete and the documentation updated, let's link here to the Account type page as an example of more complex entitlement mappings */}
```

Comment:
Same here: turn into a Markdown comment, i.e. `<!-- ... -->`

Reply:
When I tried doing this, it made the CI complain. I'm pretty sure these are comments as-is, since they don't show up here: https://developers.flow.com/next/cadence/language/access-control